### PR TITLE
P2614R2 Deprecate numeric_limits::has_denorm

### LINKED ISSUE #6108: [2023-02 LWG Motion 18] P2614R2 Deprecate numeric_limits::has_denorm

### PR DESCRIPTION
**Pull request by jensmaurer**
Fixes #6108 
Fixes cplusplus/papers#1278
Fixes cplusplus/nbballot#506

### CODE REVIEW

File: source/xrefdelta.tex [hunk 1]
```diff
@@ -110,5 +110,9 @@
 \movedxref{expos.only.func}{expos.only.entity}
 \removedxref{expos.only.types}
 
+% P2614R2 Deprecate numeric_limits::has_denorm
+\movedxref{denorm.style}{depr.numeric.limits.has.denorm}
+\removedxref{fp.style}
```
Comment (JohelEGP) [inline source/xrefdelta.tex:115]: How about having an "split reference" command. Whereas it is true that [fp.style] was removed, it seems more useful to say it was split between [round.style] and [depr.numeric.limits.has.denorm].

**Review by JohelEGP — APPROVED** [review]

File: source/xrefdelta.tex (hunk 1 above)
Comment (jensmaurer) [reply source/xrefdelta.tex:115]: What should the representation in the actual index be?

Comment (JohelEGP) [reply source/xrefdelta.tex:115]: A combination of these:

So something like
> fp.style _split_, _see_ round.style, depr.numeric.limits.has.denorm

Comment (tkoeppe) [reply source/xrefdelta.tex:115]: Pull requests welcome :-)

**Pull request merged**